CHANGELOG.md

```diff
@@ -31,7 +31,19 @@ The [1.5 release](docs/1_5_Upgrade.md) is scheduled for early July.
 
 ### Bug Fixes
 
-- Fixed case where networks using onchain permissioning could stall when the bootnodes were not validators. [\#969](https://github.com/hyperledger/besu/pull/969)
+- Added timeout to queries.  Fixes issue where `eth_getLogs` queries that were too large or too broad 
+could cause Besu to crash. [\#986](https://github.com/hyperledger/besu/pull/986)
```

Comment:
Besu can still crash/hang, that's the deeper fix we mentioned.
```suggestion
- Added timeout to queries.  Fixes issue where `eth_getLogs` queries that were too large or too broad 
never return. [\#986](https://github.com/hyperledger/besu/pull/986)
```

Reply:
Fixed and added the known issue back.  Thanks for catching my miss.